packages/pds/src/account-manager/helpers/password.ts

```diff
@@ -27,6 +27,15 @@ export const verifyAppPassword = async (
   did: string,
   password: string,
 ): Promise<AppPassDescript | null> => {
+  // Fast check for the password format
+  if (
+    password.length !== 19 ||
+    password[4] !== '-' ||
+    password[9] !== '-' ||
+    password[14] !== '-'
+  ) {
+    return null
+  }
```

Comment:
I am a bit concerned about this ending up being too strict and biting us, e.g. if the format changes in the future or has ever changed in the past.  I don't think it will harm us for for app password checks to be equally efficient to regular password checks.